modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -119,60 +129,61 @@
 			.e-n-tab-icon {
 				display: flex;
 				align-items: center;
+				flex-direction: column;
 				order: var( --n-tabs-icon-order );
+				overflow: hidden;
```

Comment:
You are cutting the icon it's bigger than its container?

Reply:
I am hidden the active/non-active icon.

The icon itself will alway take on the size of the container.